padsearch/padsearch.py

```diff
@@ -527,15 +579,17 @@ def __init__(self, bot):
         
         matched_monsters.sort(key=lambda m: m.monster_no_na, reverse=True)
 
-        msg = 'Matched {} monsters'.format(len(matched_monsters))
-        if len(matched_monsters) > 10:
-            msg += ' (limited to 10)'
-            matched_monsters = matched_monsters[0:10]
-
+        msg = []
+        for page in range(0,int(len(matched_monsters)/10)+1):
+            msg.append('Matched {} monsters'.format(len(matched_monsters)))
+            if len(matched_monsters) > 10:
+                msg[page] += ' (limited to 10)'#  Page {}'.format(page+1)
+        n = 0
         for m in matched_monsters:
-            msg += '\n\tNo. {} {}'.format(m.monster_no_na, m.name_na)
-
-        await self.bot.say(box(msg))
+            msg[int(n/10)] += '\n\tNo. {} {}'.format(m.monster_no_na, m.name_na)
+            n += 1

```

Comment:
Is this prep work for pagination? Because I don't see what this is adding at the moment.

Reply:
Yes